demos/collection/card-deck.php

```diff
@@ -47,6 +47,6 @@
     'country' => ['required' => true, 'ui' => ['form' => [Form\Control\Lookup::class, 'model' => new Country($app->db), 'placeholder' => 'Please select a country.']]],
 ];
 
-$deck = CardDeck::addTo($app, ['noRecordScopeActions' => ['request_info'], 'singleScopeActions' => ['book']]);
+$deck = CardDeck::addTo($app);
```

Comment:
I will revert this repro code not because it is solely bad, but we need it for more coverage, otherwise `noRecordScopeActions` would be untested.